R/as_rtf.R

```diff
@@ -296,7 +296,7 @@ as_rtf.gs_design <- function(
   orientation <- match.arg(orientation)
 
   method <- intersect(class(x), c("ahr", "wlr", "combo", "rd"))[1]
-  x_alpha <- max((x %>% dplyr::filter(Bound == display_bound[1]))[["Null hypothesis"]])
+  x_alpha <- max(filter(x, Bound == display_bound[1])[[colname_spannersub[2]]])
```

Comment:
Now I think we should do it the other way around, i.e., we should make `as_gt()` consistent with `as_rtf()` by hardcoding the string since it seems `"Null hypothesis"` is hardcoded in `x`. Correct me if I'm wrong.

Reply:
Forget it. I think I was wrong. Changing it will break a test.